### DISCUSSION TITLE
[Feature] Supercharge "Memories on this day" with Quick Actions

### DISCUSSION THREAD

**Discussion by mvgt13** [immich-app/immich#14817] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I love the “**Memories on this day**” Feature that shows me the pictures shot in the past years. What i used to do in the similar feature in MS OneDrive almost daily was to delete unwanted pictures and clean up my Library this way.

I would love to do the same on the fly with my immich!

So my Feature request would be to add some quick action buttons to the "Memories on this day" feature, enabling users to efficiently organize their photo library while enjoying their daily photo memories.

## Your Benefits

- **Improved Library Quality**: Gradual, bite-sized organization leads to a better-maintained photo collection over the years. The compound effect was HUGE for me.
- **Time Efficiency**: Users can organize their library during natural moments of engagement rather than requiring dedicated cleanup sessions. It will break down library organization into manageable daily tasks.
- **Increased User Engagement**: Users will open the app more frequently, combining the joy of revisiting memories with productive library management.

## Quick Action Buttons

1. Favorite Image
2. Delete Image
3. Add to Album
4. Show Metadata/Info

### Enhanced Metadata Display

- Image Folder Path
- IPTC Keywords/Tags
- Other relevant metadata for external library management

## Technical Considerations

- Quick actions can be implemented as overlay buttons or swipe actions
- Metadata display can be toggled through the info button
- Actions could support batch operations for multiple photos from the same day in a grid view

## Impact on User Experience

This enhancement transforms passive memory viewing into an active library management opportunity, without compromising the emotional experience of revisiting memories.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4652
